elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -44,6 +46,7 @@ $variables: (
   // properties
   display:         inline-block;
   position:        relative;
+  z-index:         0;
   max-width:       max-content;
   vertical-align:  middle;
   transition:      padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
```

Comment:
@LyndseyR @starryeyez024 If we add `white-space: nowrap;`, I suspect it will fix our arrow wrapping issue!! 💃